base/docs/basedocs.jl

```diff
@@ -1673,7 +1673,7 @@ ErrorException
 """
     FieldError(type::DataType, field::Symbol)
 
-An operation tried to access invalid `field` of `type`.
+An operation tried to access invalid field `field` on an object of type `type`.
```

Comment:
At least to me this isn't more clear than before, if anything it feels confusing when I am reading the same word twice in a row. The signature already has `type` as a `DataType` and the `field` is the field.